packages/compat/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"noImplicitAny": false,
+		"noImplicitThis": false
+	},
+	"files": [ "index.js" ],
+	"references": [ { "path": "../compile" } ]
```

Comment:
`compat` has a dependency on `../tannin` also. Should it be included here as well?

Reply:
This should be corrected in 340406e7b9cb3ea67ceae8165e9a329d9e99abb5